### PR TITLE
Previewer option from config

### PR DESCRIPTION
**Pull request by topless**
* `isPreviewerHidden` now gets a default value `false` from the config file.

### CODE REVIEW

File: example/configs/config.ts [hunk 1]
```diff
@@ -147,6 +147,7 @@ export const defaultConfig: JsonEditorConfig = {
       }
     }
   },
+  isPreviewerHidden: false,
```
Comment (harunurhan) [inline example/configs/config.ts]: you can remove this, since it's already default.

File: src/shared/interfaces/json-editor-config.ts [hunk 2]
```diff
@@ -46,6 +46,13 @@ export interface JsonEditorConfig {
    */
   schemaOptions?: SchemaOptions;
 
+  /**
+   * If set, it hides the preview section of the editor.
+   *
+   * - It is `false` by default.
+   */
+  isPreviewerHidden?: boolean;
```
Comment (harunurhan) [inline src/shared/interfaces/json-editor-config.ts]: can you rename it to `isPreviewerHiddenOnStart` ?

Comment (topless) [reply src/shared/interfaces/json-editor-config.ts]: I kept the property name that is used from the ui when shows/hides the previewer. There is also another option `isPreviewDisabled` so I tried not to create yet another one. But I agree is more self explained.

Comment (harunurhan) [reply src/shared/interfaces/json-editor-config.ts]: Yeah, since it's not always hidden and can be toggled later, I'd go with explicit name.

**Pull request merged**